.husky/pre-commit

```diff
@@ -0,0 +1 @@
+pnpm test
```

Comment:
This is probably too much. We can run linting/prettifying on pre-commit but running tests can take time, making contributions slow. Let's trust PR checks for tests.

Reply:
Okay 🗡️